string actionName
string gripper
string obj
string button 
---
geometry_msgs/PoseStamped[] endEffectorInfo